molecule/haproxy/verify.yml

```diff
@@ -13,6 +13,11 @@
         ansible_facts.distribution_release | lower == 'bookworm' and haproxy_version | regex_search('\\d+\\.\\d+') is version('2.4', '==') or
         ansible_facts.distribution_release | lower == 'buster' and haproxy_version | regex_search('\\d+\\.\\d+') is version('2.8', '==')
 
+    - name: "Stop play for unsupported HAProxy / Ubuntu combinations"
+      ansible.builtin.meta: "end_host"
+      when: >-
+        ansible_facts.distribution_release | lower == 'noble' and haproxy_version | regex_search('\\d+\\.\\d+') is version('2.9', '<')
```

Comment:
So, for Noble Numbat / 24.04 no HAProxy Version (2.4, 2.6, 2.8) is supported, if the run with everything lower than 2.9 is stopped?

Reply:
See above.